website/templates/project/edit_draft_registration.mako

```diff
@@ -8,22 +8,20 @@
         <div class="col-md-9">
           Edit draft registration
         </div>
-        <!-- TODO: uncomment to allow submit for review
         <div class="col-md-3" data-bind="with: draft">
           <span class="btn-group" data-bind="if: requiresApproval">
             <a data-bind="click: $root.submitForReview,
                           css: {
-                            'disable': isPendingReview
+                          'disable': isPendingReview
                           },
                           tooltip: {
-                            position: 'top',
-                            title: isPendingReview ? 'Request for review already sent' : 'Submit for review'
+                          position: 'top',
+                          title: isPendingReview ? 'Request for review already sent' : 'Submit for review'
                           }" class="btn btn-default" type="button">
-              <i class="fa fa-save"></i> Submit for review
+              <i class="fa fa"></i> Submit for review
```

Comment:
This does not seem to refer to any particular icon- should this be class `fa fa-save`?